pkg/parser/ast/sem.go

```diff
@@ -0,0 +1,1315 @@
+func (n *DropUserStmt) SEMCommand() string {
+	return DropUserCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *FlashBackDatabaseStmt) SEMCommand() string {
+	return FlashBackDatabaseCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *FlashBackTableStmt) SEMCommand() string {
+	return FlashBackTableCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *FlashBackToTimestampStmt) SEMCommand() string {
+	return FlashBackClusterCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *LockTablesStmt) SEMCommand() string {
+	return LockTablesCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *OptimizeTableStmt) SEMCommand() string {
+	return OptimizeTableCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *RecoverTableStmt) SEMCommand() string {
+	return RecoverTableCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *RenameTableStmt) SEMCommand() string {
+	return RenameTableCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *RenameUserStmt) SEMCommand() string {
+	return RenameUserCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *RepairTableStmt) SEMCommand() string {
+	return AdminRepairTableCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *TruncateTableStmt) SEMCommand() string {
+	return TruncateTableCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *UnlockTablesStmt) SEMCommand() string {
+	return UnlockTablesCommand
+}
+
+// DML Statements
+
+// SEMCommand returns the command string for the statement.
+func (n *CallStmt) SEMCommand() string {
+	return CallCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *DeleteStmt) SEMCommand() string {
+	return DeleteCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *DistributeTableStmt) SEMCommand() string {
+	return DistributeTableCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *ImportIntoStmt) SEMCommand() string {
+	return ImportIntoCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *InsertStmt) SEMCommand() string {
+	if n.IsReplace {
+		return ReplaceCommand
+	}
+	return InsertCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *LoadDataStmt) SEMCommand() string {
+	return LoadDataCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *NonTransactionalDMLStmt) SEMCommand() string {
+	return BatchCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *SelectStmt) SEMCommand() string {
+	return SelectCommand
+}
+
+// SEMCommand returns the command string for the statement.
+func (n *SetOprStmt) SEMCommand() string {
+	return UnionCommand
```

Comment:
Seems this is wrong?

Reply:
Fixed. I used `SET OPERATION` instead as it may contain `UNION`, `INTERSECT` and `EXCEPT`.